shell/core/plugin-routes.ts

```diff
@@ -34,70 +22,25 @@ export class PluginRoutes {
     });
   }
 
-  // Ensure we put back any routes that the plugin that is being uninstalled added
-  public uninstall(plugin: any) {
-    // List of routes we need to restore
-    const restore: RouteInfo[] = [];
-
-    Object.keys(this.replacedRoutes).forEach((routeName) => {
-      const info = this.replacedRoutes[routeName];
-
```

Comment:
Uninstall really shouldn't be necessary. We require the user refresh the page on uninstall.

Reply:
I looked at the code and couldn't quite trace it back correctly, but my main concern here is what happens on a extension `upgrade`? Does it do an uninstall of the previous version of the extension? If so, then if we aren't removing the previous extension routes, do they duplicate?

Reply:
I tested the change when updating and we still prompt the user to reload the page so I think it's fine.